index.html

```diff
@@ -4,10 +4,49 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
-    
+<body class="flex_column">
+    <header>
+        <nav class="flex">
+            <div class="logo flex_column">
+                <a href=""><img src="assets/celec.png" alt="logo"></a>
+                <p>My logo</h2>
+            </div>
+            <a href="#" class="burgeur flex"><img src="assets/Icon - Menu.png" alt="menu"></a>
+        </nav>
+    </header>
+
+    <main>
+        <section id="Home">
+            <div class="headline flex_column">
+                <div class="description">
+                    <h1>Hey There <br> I'm Alex.</h1>
+                    <h2>I'm a Software Developer</h2>
+                    <p>I can help you build a product , feature or website.
+                         Look through some of my work and experience! 
+                         If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                </div>
+                <ul class="flex">
+                    <li><a href="https://github.com/BenM4k"><img src="assets/github.svg" alt=""></a></li>
+                    <li>
+                        <a href="https://www.linkedin.com/in/b%C3%A9n%C3%A9dict-makomo-885856248/">
+                            <img src="assets/Linkedin icon.svg" alt="">
+                        </a>
+                    </li>
+                    <li><a href="https://angle.co"><img src="assets/angel list.svg" alt=""></a></li>
+                    <li><a href="https://twitter.com"><img src="assets/twitter.svg" alt=""></a></li>
+                    <li><a href="https://medium.com"><img src="assets/medium.svg" alt=""></a></li>
+                </ul>
+                <img src="assets/Group.png" alt="bg" class="bg_img">
```

Comment:
- [ ] As the previous reviewer suggested, please, ensure that your `img` tag always has an `alt` attribute description, this is good for accessibility reasons, for example, for a visually impaired individual or a slow internet user.
- [ ] [OPTIONAL] kindly consider using the [relative image path](https://www.w3schools.com/html/html_filepaths.asp). For example `<img src="../image_store/Part2.jpg" alt="Dress Picture"/>`.